move_base/src/move_base.cpp

```diff
@@ -383,10 +383,45 @@ namespace move_base {
     controller_costmap_ros_->getCostmap()->setConvexPolygonCost(clear_poly, costmap_2d::FREE_SPACE);
   }
 
-  bool MoveBase::clearCostmapsService(std_srvs::Empty::Request &req, std_srvs::Empty::Response &resp){
+  bool MoveBase::clearCostmapsService(std_srvs::Trigger::Request &req, std_srvs::Trigger::Response &resp){
```

Comment:
Since this is a separate git repo, we need to figure out the best way to incorporate this into our build process.  Could be a separate workspace, submit the patch back to the ros repository and switch back if/when they incorporate.  Or add our repo/branch as a submodule (https://git-scm.com/book/en/v2/Git-Tools-Submodules).